homeassistant/helpers/script.py

```diff
@@ -420,6 +421,41 @@ async def async_stop(self) -> None:
         self._stop.set()
         await self._stopped.wait()
 
+    def _handle_exception(
+        self, exception: Exception, continue_on_error: bool, log_exceptions: bool
+    ) -> None:
+        if not isinstance(exception, (_AbortScript, _StopScript)) and log_exceptions:
+            self._log_exception(exception)
+
+        if not continue_on_error:
+            raise exception
+
+        # An explicit request to stop the script has been raised.
+        if isinstance(exception, _StopScript):
+            raise exception
+
+        # These are incorrect scripts, and not runtime errors that need to
+        # be handled and thus cannot be stopped by `continue_on_error`.
+        if isinstance(
+            exception,
+            (
+                vol.Invalid,
+                exceptions.TemplateError,
+                exceptions.ServiceNotFound,
+                exceptions.InvalidEntityFormatError,
+                exceptions.NoEntitySpecifiedError,
+                exceptions.ConditionError,
+            ),
+        ):
+            raise exception
+
+        # Only Home Assistant errors can be ignored.
+        if not isinstance(exception, exceptions.HomeAssistantError):
+            raise exception
+
+        # Ignore the error
+        script_execution_set("error")
```

Comment:
Why set it to error if we're ignoring the error? Also, this would be overridden when the script finishes ?

Reply:
Right, this does absolutely nothing 😬 
Removing the line.